src/data/faq.json

```diff
@@ -87,6 +87,130 @@
+                            "Contact persons receive a warning in their Corona-Warn-App about an encounter on a day with low (green tile) or increased risk (red tile). Whether the warning was based on a rapid test or on a PCR test is not displayed in the app."
+
+                        ]
+                    },
+                    {
+                        "title": "EFGS integration? Are positive results shared in other countries?",
+                        "anchor": "rat_efgs",
+                        "active": true,
+                        "textblock": [
+                            "Warnings based on a rapid test result are not shared with other EU countries via the EFGS (European Federation Gateway Service), including Switzerland, as the EFGS terms of use do not provide for rapid test warnings to be shared."
+                        ]
+                    },
+                    {
+                        "title": "What is the difference between PCR tests, rapid antigen tests and self-tests?",
+                        "anchor": "rat_differences_pcr",
+                        "active": true,
+                        "textblock": [
+                            "PCR tests are the 'gold standard' among corona tests. Sampling is performed by medical personnel - evaluation by laboratories.",
+                            "Rapid antigen tests: Have their name because the result is available quickly. They can only be performed by trained personnel. Similar to the PCR test, a nasal or throat swab is taken. However, in contrast to the PCR test, the evaluation takes place directly on site. Since March 8, 2021, everyone is entitled to at least one rapid test per week.",
+                            "Self-tests: Have their name because anyone can do these tests themselves, for example at home. The self-tests are intended for use by private individuals. For this purpose, sample collection and evaluation must be appropriately simple. The test can be performed with a nasal swab or saliva, for example. The Federal Institute for Drugs and Medical Devices, together with the Paul Ehrlich Institute, checks the quality and significance of the tests.",
+                            "Rapid and self-tests have a higher error rate compared to PCR tests. Therefore, after each positive rapid and self-test, a PCR test should always be done for confirmation."
+                        ]
+                    },
+                    {
+                        "title": "Can I share my positive rapid antigen test result via the TAN hotline?",
+                        "anchor": "rat_tan_hotline",
+                        "active": true,
+                        "textblock": [
+                            "It is not possible to share a positive rapid test result via the TAN hotline. Only rapid test results communicated via the app have been performed according to the dual control principle and by trained personnel, and can therefore be shared."

```

Comment:
"via the app have been performed" => "via the app and which have been performed"